source/NVDAObjects/window/excel.py

```diff
@@ -1686,7 +1696,9 @@ def script_reportComment(self,gesture):
 	@script(
 		# Translators: the description  for a script for Excel
 		description=_("Opens the note editing dialog"),
-		gesture="kb:shift+f2")
+		gesture="kb:shift+f2",

```

Comment:
Should this script be remappable in the first place? While an argument can be made about users who want to use different combination of keys to show the editing dialog, someone may also remove the default assignment and end up with an inaccessible editing dialog. Are we fine with that risk?